nixops/backends/none.py

```diff
@@ -109,6 +109,7 @@ def _check(self, res):
             res.exists = False
             return
         res.exists = True
+        # TODO: Use a better method that works with bastion hosts
         res.is_up = nixops.util.ping_tcp_port(self.target_host, self.ssh_port)
```

Comment:
I think we're guaranteed to have a working SSH, so let's just use the same "ssh and run `true`" strategy as in other places. @aanderse mentioned some backends might not need to use SSH, but that is a big enough thing to be a whole feature request and "epic".